nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -186,7 +196,10 @@ in
           WorkingDirectory = cfg.dataDir;
           User = cfg.user;
           Group = cfg.group;
-          EnvironmentFile = [ environmentFile ];
+          EnvironmentFile = [
+            cfg.settingsFile
+            environmentFile
```

Comment:
In the description of `services.healthchecks.settings` you said variables from `settingsFile` have priority, however you made it here so `settings` have priority over `settingsFile`.

> If the same variable is set twice from these files, the files will be read in the order they are specified and the later setting will override the earlier setting.
[source](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#EnvironmentFile=:~:text=If%20the%20same%20variable%20is%20set%20twice%20from%20these%20files%2C%20the%20files%20will%20be%20read%20in%20the%20order%20they%20are%20specified%20and%20the%20later%20setting%20will%20override%20the%20earlier%20setting.)

Reply:
Right... fixed now.